compiler/src/test/resources/bugs/apps_994_v2.wdl

```diff
@@ -0,0 +1,93 @@
+version 1.1
+
+task reuse_multiply {
+	input {
+		Int first
+		Int second
+	}
+
+	command <<< >>>
+
+	runtime {
+		container: "ubuntu:20.04"
+	}
+
+	Int product = first * second
+
+	output {
+		Int out = product
+	}
+}
+
+task reuse_print {
+	input {
+		String str
+	}
+
+	command <<<
+	set -euxo pipefail
+	echo ~{str} > reuse_print.txt
+	>>>
+
+	runtime {
+		container: "ubuntu:20.04"
+	}
+
+	output {
+		File out = "reuse_print.txt"
+	}
+}
+
+workflow reuse {
+
+	# Frag applet via sub-expression
+	# declaration of inputs is dofferent from V1. Therefore this frag should be rebuilt
```

Comment:
*different

Reply:
fixed in [98647f0](https://github.com/dnanexus/dxCompiler/pull/291/commits/98647f0cb1881747656c96096189f220c7a0b1d6)